frontend/src/components/AboutMe/ProfileCard.jsx

```
@@ -17,33 +21,127 @@
+          <div className="absolute top-28 w-60 bg-white border border-gray-300 rounded-xl shadow-lg p-4 z-10 flex flex-col items-center">
+            <img
+              src={previewURL}
+              alt="New Preview"
+              className="w-20 h-20 rounded-full object-cover mb-3 border"
+            />
+            <p className="text-sm text-gray-700 font-medium mb-2">Use this photo?</p>
+            <div className="flex gap-3">
+              <button
+                className="bg-blue-600 text-white px-4 py-1 rounded-full text-sm hover:bg-blue-700 transition"
+                onClick={handleUpload}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)